index.html

```diff
@@ -340,3 +340,22 @@ <h3 id="render" data-export="" data-dfn-type="dfn">
 and will create user agents to represent those preferences on the web user's behalf.
   </p>
 </section>
+
+<section>
+  <h3 id="centralization" data-export="" data-dfn-type="dfn">
+The web is decentralized
+  </h3>
+  <p>
+    The web is a platform designed to be used and operated by as wide
+    a range of individuals and orgaizations as possible. 
+    We will not create web technologies that encourage centralization,
+    by favoring large or powerful organizations, to the detriment of smaller
+    organizations. We include in this concern both technologies
+    that explicitly promote centralization (e.g., systems that rely on
+    a single or small number of "trusted parties" to collect or pre-process data),
```

Comment:
The DNS is hosted by a 'small number of "trusted parties"'. Are you suggesting that the Web shouldn't use it?

There are only three browser engines; does that represent a 'small number of "trusted parties"'?

Does this mean that Google Safe Browsing is centralized?

Reply:
> The DNS is hosted by a 'small number of "trusted parties"'. Are you suggesting that the Web shouldn't use it?

No, i dont intend or think DNS would be covered by this wording.  I can stand up a DNS server at home and it'll work just as well. But it would be very bad for a web standard to say "step 3: talk to Google, CF or Comcast's DNS servers".

> There are only three browser engines; does that represent a 'small number of "trusted parties"'?

No I didn't intend browser engines to be covered here. I'm happy to revise the text to make that clearer.  A browser engine isn't a trusted party, its auditable (or at least an implementation could be auditable, even if not all are).  But, it would be bad if a standard was written to require talking to software or a service that was designed to be unauditable ( widevine…), for example.

> Does this mean that Google Safe Browsing is centralized?

Google safe browsing is absolutely centralized. I think it'd be very bad to have a W3C standard that required talking to Google Safe Browsing servers to work correctly. Its a useful and valuable system, and I'm glad it exists, but it doesn't seem like a good model for building open standards for the Web.

Reply:
re DNS, @mnot I realize I might have misunderstood your comment.  Were you referring to root servers, or resolvers?  My comment was about the latter, but on second thought I see you might have meant the former.

If you meant root servers, yea, thats a tricky one, I take your point. Its unappealing, but maybe its least bad. I could try and emphasize "unless necessary" or "unless no other alternatives exist" or similar (though i appreciate that direction has its own downsides too)

Reply:
Indeed, I meant the root servers.